textkit/install_dependencies.py

```diff
@@ -0,0 +1,50 @@
+import os
+import click
+import nltk
+from textkit.utils import read_tokens, output
+
+
+@click.command('install_dependencies')
+@click.option('--path', 
+              help='Separator between token and count in output.',
+              show_default=True)
```

Comment:
is there going to be a default? I see show_default is true - but I don't see a default.